__tests__/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,30 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Index should be able to resolve the same dependencies 1`] = `
```

Comment:
Just to capture what the API should be like will be helpful for catching the public API changes.